packages/trader/src/Modules/Trading/Containers/trade-params-mobile.tsx

```diff
@@ -78,10 +78,11 @@ const reducer = (state: TReducer, payload: Partial<TReducer>) => {
     };
 };
 
-const makeGetDefaultDuration = (trade_duration: number, trade_duration_unit: string) => (duration_unit: string) =>
-    trade_duration_unit === duration_unit
+const makeGetDefaultDuration = (trade_duration: number, trade_duration_unit: string) => (duration_unit: string) => {
+    return trade_duration_unit === duration_unit
```

Comment:
We can get rid of `{return}`

Reply:
oh opps i was trying something and got left in there